android/src/main/java/bbct/android/common/activity/BaseballCardList.java

```diff
@@ -296,8 +296,14 @@ protected void applyFilter(Bundle filterParams) {
 
         if (this.filterParams == null) {
             this.emptyList.setText(R.string.start);
+            String emptyListTitle = this.getString(R.string.start);
+            String title = this.getString(R.string.bbct_title, emptyListTitle);
+            this.getActivity().setTitle(title);
         } else {
             this.emptyList.setText(R.string.empty_list);
+            String emptyListTitle = this.getString(R.string.empty_list);
+            String title = this.getString(R.string.bbct_title, emptyListTitle);
+            this.getActivity().setTitle(title);
```

Comment:
`R.string.start` and `R.string.empty_list` are only intended for use in the `ListView` when it contains no items. This should not be displayed in the activity's title.

Reply:
Can I put the empty title, and start title to a different string resource? or should I just remove this part?

Reply:
I recently removed this one.